docker/pravega/Dockerfile

```diff
@@ -7,10 +7,14 @@
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
-FROM openjdk:8-jre
+FROM openjdk:8-jre-alpine
 
-RUN apt-get update && apt-get install -y -q rpcbind nfs-common \
-    && rm -rf /var/lib/apt/lists/*
+RUN apk add --update \
+    rpcbind \
+    nfs-utils \
+    libc6-compat \
```

Comment:
Did not understand the reason for this change?

Reply:
Since we are changing the base image to Alpine, we need to use the `apk` package manager instead of `apt`. Package names for Alpine may differ to Debian packages. In the case of the Pravega image, we made the following changes:

- `rpcbind` Alpine package replaces the `rpcbind` Ubuntu package
- `nfs-utils` Alpine package replaces the `nfs-common` Ubuntu package
- `libc6-compat` Alpine package contains the `ld-linux-x86-64.so.2` library, which is a dependency of RocksDB. `libc6` was already installed in the previous base image.